frontends/ol-components/src/components/Chips/Chip.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react"
+import type { Meta, StoryObj } from "@storybook/react"
+import Chip from "@mui/material/Chip"
+import Stack from "@mui/material/Stack"
+import { fn } from "@storybook/test"
+import { ChipLink } from "./ChipLink"
+import { withRouter } from "storybook-addon-react-router-v6"
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday"
+
+const COLORS = ["default", "primary", "secondary"] as const
+const VARIANTS = [
+  {
+    variant: "filled",
+    label: "Filled",
+  },
+  {
+    variant: "outlined",
+    label: "Outlined",
+  },
+] as const
+
+const meta: Meta<typeof Chip> = {
+  title: 'smoot-design/Chip ("Pill")',
+  component: Chip,
+  render: (args) => {
+    return (
+      <Stack gap={1}>
+        {VARIANTS.map(({ variant, label }) => (
+          <Stack key={variant} direction="row" gap={2}>
+            {COLORS.map((color) => (
+              <Chip
+                key={color}
+                variant={variant}
+                color={color}
+                label={label}
+                {...args}
+              />
+            ))}
+          </Stack>
+        ))}
+        {VARIANTS.map(({ variant, label }) => (
+          <Stack key={variant} direction="row" gap={2}>
+            {COLORS.map((color) => (
+              <Chip
+                size="large"
+                variant={variant}
+                key={color}
+                color={color}
+                label={label}
+                {...args}
+              />
+            ))}
+          </Stack>
+        ))}
+      </Stack>
+    )
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof Chip>
+
+export const Variants: Story = {}
+
+export const Buttons: Story = {
+  args: {
+    onClick: fn(),
+  },
+}
+
+export const Deleteable: Story = {
```

Comment:
Should these be removed in the Storybook UI when deleted?

Reply:
No. It just logs the onDelete events:

<img width="525" alt="Screenshot 2024-04-22 at 4 09 41 PM" src="https://github.com/mitodl/mit-open/assets/9010790/a99b4f9d-ba9e-4b57-a26d-560fba68cb71">